digiwf-gateway/src/main/resources/application.yml

```diff
@@ -53,3 +53,6 @@ management:
     env:
       enabled: true
 info.application.name: ${spring.application.name}
+logging:
+  level:
+    root: DEBUG
```

Comment:
Das ist auch sicher Testzweck und sollte nicht reinkommen in den allgemeinen Properties

Reply:
guter Punkt. wurde gelöscht